File: DotNet/CesiumLanguageWriter/Advanced/ICesiumReferenceListOfListsValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of lists of references to other properties.
    /// </summary>
    public interface ICesiumReferenceListOfListsValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of lists of references.
        /// </summary>
        /// <param name="references">The list of lists of references.</param>
        [CSToJavaUseWildcardGenerics]
        void WriteReferences(IEnumerable<IEnumerable<Reference>> references);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumReferenceListOfListsValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of lists of references to other properties.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumReferenceListOfListsValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of lists of references.
    * @param references The list of lists of references.
    */
    void writeReferences(Iterable<? extends Iterable<? extends Reference>> references);
}